src/lib/relationship_graph.cc

```diff
@@ -702,12 +740,25 @@ void dng::RelationshipGraph::CreatePeelingOps(
 
     auto edge_types = get(boost::edge_type, pedigree_graph);
     auto lengths = get(boost::edge_length, pedigree_graph);
+    auto gender  = get(boost::vertex_gender, pedigree_graph);
 
     ResetFamilyInfo();
+    for (std::size_t i = first_founder_; i < first_nonfounder_; ++i) {
```

Comment:
I think the hack k==i+1 is true all the time.
But, let's redo this part later, after all inheritance models are implemented.
Added PR_NOTE and test/refactor this later.